core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -438,6 +438,16 @@ public QueryAssert containsAll(MaterializedResult expected)
             });
         }
 
+        @CanIgnoreReturnValue
+        public QueryAssert hasOutputNames(List<String> expectedNames)
```

Comment:
maybe vararg instead: `public QueryAssert hasOutputNames(String... columnNames)`

then the assertion will be more readable:

```
assertThat(query("SELECT * FROM TABLE(system.query(query => 'SELECT name AS region_name FROM tpch.region WHERE regionkey = 0'))"))
                .hasColumnNames("region_name")
                .matches("VALUES CAST('AFRICA' AS VARCHAR)");

```